extras/nginx-mtls-auth/README.md

```diff
@@ -0,0 +1,39 @@
+# Sample of using securing Model Server with nginx
+
+Dockerfile and scripts in this directory are an example of using Nginx mTLS module to implement authentication and authorization of OpenVINO Model Server.
+
+This can secure both GRPC and REST endpoints. Model Server will present server certificate, and allow connection only from clients who perform full TLS handshake (successful client certificate authentication).
+
+WARNING: Those contain certificate generation automation for development and testing purposes. Do not use those in production - follow best practices of your organization. For ensuring "fast fail", certificates generated here will expire after a single day.
+
+WARNING: Only minimal nginx configuration is presented. Your are responsible for developing and using a secure configuration.
```

Comment:
It is fully functional configuration. It should be sufficient to add a note that most of the ngnix configuration settings have the default settings. Review them and adjust them according to your organization policies.

Reply:
Changed.